hooks/useAvalancheForecastFragment.ts

```diff
@@ -17,8 +17,10 @@ export const useAvalancheForecastFragment = (center_id: string, forecast_zone_id
     const {data} = await axios.get(url, {
       params: {
         avalanche_center_id: center_id,
-        date_start: format(sub(date, {days: 1}), 'y-MM-dd'),
-        date_end: format(add(date, {days: 1}), 'y-MM-dd'),
+        forecast_zone_id,
+        // TODO(brian): remove this hack of adding/subtracting two days, which works around issues converting between local day and UTC day
```

Comment:
`date-fns` can convert between time-zones, should we just do that instead of this?

Reply:
yeah, I just want to write a test to make sure the fix is right, so I was hoping to unblock this PR and then take a careful pass. 

I _think_ the issue is the format string implicitly says "give me the date in my current TZ" but want to be sure - and also want to check for any other place we might be doing this.

Reply:
/shrug

I don't know that we have per-day deadlines that you need to hit, or obversely the cost of this "block" is small :)

FWIW I think we could also ask for clarification for what, specifically, the API is applying these date ranges to - the date at which the forecast was published? Last updated? The dates on which the forecast is valid?

Reply:
Tracked in https://github.com/stevekuznetsov/avalanche-forecast/issues/45